### ISSUE TITLE
Add option to disable resourcepack distribute

### ISSUE THREAD

**Issue by TetraTheta** [MC-Machinations/VanillaTweaks#14]
I am using this plugin, VanillaTweaks with Force Resourcepack plugin. Since FRP tries to push resource pack while VT also tries to push its own resource pack to client, client loads twice.  
I think only way to fix this double loading issue, VT should provide option of 'Disable Resource Pack Distribute' so that server owner can manually merge those resource packs into one to prevent multiple resource pack loading issue.

TL;DR  
VT tries to push its own Resource Pack while other plugin/minecraft tries too. Please provide option for disabling VT's Resource Pack distribution.
